client/mksubvolume.cc

```diff
@@ -258,6 +267,33 @@ find_filesystem(MntTable& mnt_table)
     }
 }
 
+struct TmpMountpoint {
+    unique_ptr<char[]> mountpoint;
+    const libmnt_fs* fs;
+    TmpMountpoint(const string tmpMountpoint, const libmnt_fs* libmntfs, const string& subvol_opts)
```

Comment:
tmpMountpoint should be passed by reference.

Reply:
Done.